### PR TITLE
Update espressif docs

### PR DESCRIPTION
**Pull request by C47D**
Updates to the Espressif documentation to keep up to date with the latest changes on the `lv_port_esp32` repo, sent to have some feedback from yall.

### CODE REVIEW

File: get-started/espressif.md [hunk 1]
```diff
@@ -4,11 +4,23 @@
 ```
 
 # Espressif (ESP32)
-LVGL includes a Kconfig file so it can be directly used as an ESP-IDF v4 component.
+Since v7.7.1 LVGL includes a Kconfig file, so LVGL can be used as an ESP-IDF v4 component.
 
-# Get the LVGL library
+# Get the LVGL demo project for ESP32
```
Comment (kisvegabor) [inline get-started/espressif.md:9]: It should be 
```suggestion
## Get the LVGL demo project for ESP32
```

File: get-started/espressif.md [hunk 2]
```diff
@@ -4,11 +4,23 @@
...
 
-When using the ESP-IDF v4 framework from Espressif you can add external libraries as "components". This components can be located inside a directory named "components" on your project root directory.
+We've created [lv_port_esp32](https://github.com/lvgl/lv_port_esp32), a project using ESP-IDF and LVGL to show one of the demos from [lv_examples](https://github.com/lvgl/lv_examples).
+You are able to configure the project to use one of the many supported display controllers, see [lvgl_esp32_drivers](htpps://github.com/lvgl/lvgl_esp32_drivers) for a complete list
+of supported display and indev (touch) controllers.
+
+# Use LVGL in your ESP32 project
```
Comment (kisvegabor) [inline get-started/espressif.md:15]:
```suggestion
## Use LVGL in your ESP32 project
```

File: get-started/espressif.md [hunk 3]
```diff
@@ -4,11 +4,23 @@
...
+
+## Prerequisites
```
Comment (kisvegabor) [inline get-started/espressif.md:17]:
```suggestion
### Prerequisites
```

File: get-started/espressif.md [hunk 4]
```diff
@@ -4,11 +4,23 @@
...
+
+ESP-IDF v4 framework is the suggested version to use.
+
+## Get LVGL
```
Comment (kisvegabor) [inline get-started/espressif.md:21]:
```suggestion
### Get LVGL
```

File: get-started/espressif.md [hunk 5]
```diff
@@ -18,4 +30,18 @@ git submodule add https://github.com/lvgl/lvgl.git components/lvgl
 
 The above command will clone LVGL's main repository into the `components/lvgl` directory. LVGL includes a `CMakeLists.txt` file that sets some configuration options so you can use LVGL right away.
 
-When you are ready to configure LVGL launch the configuration menu with `idf.py menuconfig` on your project root directory.
+When you are ready to configure LVGL launch the configuration menu with `idf.py menuconfig` on your project root directory, go to `Component config` and then `LVGL configuration`.
+
+# Use lvgl_esp32_drivers in your project
```
Comment (kisvegabor) [inline get-started/espressif.md:35]:
```suggestion
## Use lvgl_esp32_drivers in your project
```

File: get-started/espressif.md [hunk 6]
```diff
@@ -18,4 +30,18 @@ git submodule add https://github.com/lvgl/lvgl.git components/lvgl
...
+
+You are suggested to add `lvgl_esp32_drivers` as a "component". This component can be located inside a directory named "components" on your project root directory.
+
+When your project is a git repository you can include `lvgl_esp32_drivers` as a git submodule:
+
+```c
+git submodule add https://github.com/lvgl/lvgl_esp32_drivers.git components/lvgl_esp32_drivers
+```
+
+## Support for ESP32-S2
```
Comment (kisvegabor) [inline get-started/espressif.md:45]:
```suggestion
### Support for ESP32-S2
```

**Review by kisvegabor — COMMENTED** [review]
Thanks, Carlos! It looks good!

I only saw some issues with the levels of the headings. 
(See the comments)

**Comment by C47D** [comment]
@kisvegabor thanks for the feedback, I've applied the suggestions. Ready to merge!

**Comment by kisvegabor** [comment]
Thanks!

**Pull request merged**